tests/test_jwt.py

```diff
@@ -59,7 +59,7 @@ def test_sign_and_verify(self):
     self._check_sign_and_verify('privatekey.%s' % self.format)
 
   def test_sign_and_verify_from_converted_pkcs12(self):
-    """Tests that following instructions to convert from PKCS12 to PEM works."""
```

Comment:
Docstrings on tests break the output of nose tests (and are generally frowned upon).

I can leave it out of this commit, I just noticed it because it showed up in a Py3.4 failure I fixed before adding

```python
      if isinstance(signature, six.text_type):
        signature = signature.encode('utf-8')
```

to `OpenSSLVerifier.verify`.